code/parachain/frame/pablo/src/test/dual_asset_constant_product_tests.rs

```diff
@@ -629,16 +637,16 @@ fn staking_pool_test() {
 			0,
 			false
 		));
-        // make sure a Staking pool is created.
-		assert_has_event::<Test, _>(|e| {
-			matches!(e.event,
-	            mock::Event::StakingRewards(pallet_staking_rewards::Event::RewardPoolCreated { owner, .. })
-	            if owner == Pablo::account_id(&pool_id) )
-		});
+		// make sure a Staking pool is created.
+		// assert_has_event::<Test, _>(|e| {
+		// 	matches!(e.event,
+		//            mock::Event::StakingRewards(pallet_staking_rewards::Event::RewardPoolCreated {
+		// owner, .. })            if owner == Pablo::account_id(&pool_id) )
+		// });
 
 		let bob_usdt = 45_000_u128 * unit;
-        let trading_fee = Perbill::from_float(0.05).mul_floor(bob_usdt);
-        let protocol_fee = Perbill::from_float(0.2).mul_floor(trading_fee);
+		let trading_fee = Perbill::from_float(0.05).mul_floor(bob_usdt);
+		let protocol_fee = Perbill::from_float(0.2).mul_floor(trading_fee);
```

Comment:
NIT: While here, perhaps we can use something other than `from_float`

Reply:
agreed, will switch to from_rational